playbooks/provisioner/foreman/cleanup.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
I talked to @tkammer yesterday and it was decided to to leave the current work flow as is for now and we'll refactor/change it later